tbd-fe/src/components/JobListing/JobListing.scss

```diff
@@ -1,12 +1,13 @@
 @import '../../assets/variables.scss';
 
 .job {
-    box-shadow: 1px 1px 8px #B2D1E4;
-    height: 421px;
+    background-color: white;
+    box-shadow: 7px 7px 8px #B2D1E4;
+    height: 270px;
```

Comment:
I like how this is smaller now, it looked a little off with all that white space. Is it responsive at smaller screens though? I'm wondering if it might start crowding when it gets smaller. If so we might need to do a `overflow:none` or `overflow: scroll`

Reply:
It isnt responsive yet. I have a seperate branch for the responsiveness of the cards. This is just for the basic design.

Reply:
Okay, let's work on that. Glad we're thinking about!